README.md

```diff
@@ -73,4 +75,99 @@ Django-major-event-log was developed at the UNT Libraries.
 Contributors:
 
 * [Gio Gottardi](https://github.com/somexpert)
+* [Damon Kelley](https://github.com/damonkelley)
+
+
+Developing/Testing
+-------------------------
+
+#### Requirements ####
+
+- Django == 1.8
+- Python >= 2.7
+
+I suggest doing the following within a virtual environment. To do so, you will
```

Comment:
Yeah I was wondering if I should go into the virtual environment stuff or not. I can cut it. Make the README that much shorter (and by extension, sweeter).